WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -87,84 +97,121 @@ function DelegatesOfRegion({ activeSeniorDelegate, delegates, isAdminMode }) {
+    <div className="container">
+      <Header as="h1">{I18n.t('delegates_page.title')}</Header>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey="about.structure.delegates_html"
+          options={{ see_link: '' }}
+        />
+      </p>
+      <p>
+        <I18nHTMLTranslate
+          i18nKey="delegates_page.acknowledges"
+        />
+      </p>
+      {canViewDelegateMatters && (
+      <Checkbox
+        label="Enable admin mode"
+        toggle
+        checked={adminMode}
+        onChange={(__, { checked }) => setAdminMode(checked)}
+      />
+      )}
+      <Grid container>
+        <Grid.Column only="computer" computer={4}>
+          <Header>{I18n.t('delegates_page.regions')}</Header>
+          <Menu vertical>
+            {seniorDelegates.map((seniorDelegate) => (
+              <Menu.Item
+                key={`region-${seniorDelegate.id}`}
+                name={seniorLocationToRegion(seniorDelegate.location || '')}
+                active={activeSeniorDelegate === seniorDelegate}
+                onClick={() => {
+                  setActiveSeniorDelegate(seniorDelegate);
                 }}
-              />
-            </Grid.Row>
-            <Grid.Row style={{ overflowX: 'scroll' }}>
-              <DelegatesOfRegion
-                activeSeniorDelegate={activeSeniorDelegate}
-                delegates={delegates}
-                isAdminMode={isEditVisible}
-              />
-            </Grid.Row>
-          </Grid>
-        </Segment>
-      </Grid.Column>
-    </Grid>
+              >
+                {/* The 'name' shorthand above can populate the
+                label, but it sanitizes & signs :( */}
+                {seniorLocationToRegion(seniorDelegate.location || '')}
+              </Menu.Item>
+            ))}
+          </Menu>
+        </Grid.Column>
+
+        <Grid.Column stretched computer={12} mobile={16} tablet={16}>
+          <Segment>
+            <Grid container centered>
+              <Grid.Row only="computer">
+                <Header>{seniorLocationToRegion(activeSeniorDelegate.location || '')}</Header>
+              </Grid.Row>
+              <Grid.Row only="tablet mobile">
+                <Dropdown
+                  inline
+                  options={seniorDelegates.map((seniorDelegate) => ({
+                    key: `senior-delegate-${seniorDelegate.id}`,
+                    text: seniorLocationToRegion(seniorDelegate.location || ''),
+                    value: seniorDelegate.id,
+                  }))}
+                  value={activeSeniorDelegate.id}
+                  onChange={(event, { value }) => {
+                    setActiveSeniorDelegate(
+                      seniorDelegates.find((seniorDelegate) => seniorDelegate.id === value),
+                    );
+                  }}
+                />
+              </Grid.Row>
+              {/* TODO: Fix Senior Delegate ribbon CSS for tablet and mobile view,
+            and enable this component for all devices */}
```

Comment:
Is this going to be done in this PR? If not, is there an issue for it?

Reply:
Won't be done in this PR, as it requires some CSS changes and it might take some time for me.  Maybe somebody else can take it or I'll do it in long future if nobody take it up.

Reply:
Sounds good, I just want to make sure there's an issue on GitHub so that the TODO doesn't get forgotten.

Reply:
Done, https://github.com/thewca/worldcubeassociation.org/issues/8435.